storage/storage_test.go

```diff
@@ -631,15 +631,17 @@ func TestWritePartition(t *testing.T) {
 			}
 		}
 		bds := []*BlockDevice{bd}
-
-		found := FindAdvancedInstallTargets(bds)
-		if len(found) > 0 {
-			t.Fatalf("Should not have found any advanced targets %+v", found)
-		}
-
-		if !AdvancedPartitionsRequireEncryption(bds) {
-			t.Fatalf("Advanced targets should require encryption")
-		}
+		/*
+			found := FindAdvancedInstallTargets(bds)
+			if len(found) > 0 {
```

Comment:
We should leave these checks, but the logic now needs to be reversed.
```suggestion
			if len(found) == 0 {
```

Reply:
Done